README.md

```diff
@@ -212,6 +230,63 @@ $ curl localhost:9200/_cat/indices
 green open ks-logstash-log-2021.04.06 QeI-k_LoQZ2h1z23F3XiHg  5 1 404879 0 298.4mb 149.2mb
 ```
 
+### Fluentd 
+
+Fluentd acts as a forward log layer to collect logs from fluentbit and other Apps. 
+
+#### Collect logs from Fluentbit
+
+##### Enable Fluentbit forward plugin
+
+At first, we should enable the forward plugin in fluentbit to send logs to fluentd.
+
+```shell
+kubectl apply -f manifests/fluentd/fluentbit-output-forward.yaml
+```
+
+##### Fluentd ClusterFluentdConfig example
+
+```shell
+kubectl apply -f manifests/fluentd/fluentd-cluster-cfg-output-es.yaml
+```
+
+##### Fluentd FluentdConfig example
+
+```shell
+kubectl apply -f manifests/fluentd/fluentd-namespaced-cfg-output-es.yaml
+```
+
+##### Fluentd mixed FluentdConfig and ClusterFluentdConfig example
+
+```shell
+kubectl apply -f manifests/fluentd/fluentd-mixed-cfgs-output-es.yaml
+```
+
+##### Fluentd ClusterOutput and Output example in multi-tenant scenario
+
+```shell
+kubectl apply -f manifests/fluentd/fluentd-mixed-cfgs-multi-tenant-output.yaml
+```
+
+##### Fluentd outputs to kafka/es example
+
+```shell
+kubectl apply -f manifests/fluentd/fluentd-cluster-cfg-output-kafka.yaml
+kubectl apply -f manifests/fluentd/fluentd-cluster-cfg-output-es.yaml
+```
+
+##### Fluentd buffer example
```

Comment:
In line 241, you wrote `At first`. I'm wondering what's the logic here?

Reply:
Fluent bit sends log to fluentd with the inner forward plugin, and fluentd also needs to use the inner forward plugin to accept the input logs.

Reply:
So, each level-5 title in my comment here stands for an option that users can choose to run the command if they want to enable the inner forward plugin to accept the input logs?

Reply:
See below:

##### Enable Fluent Bit forward plugin

At first, we should enable the forward plugin in Fluent Bit to send logs to Fluentd.

```shell
kubectl apply -f manifests/fluentd/fluentbit-output-forward.yaml
```

And secondly, Fluentd also needs to use the forward input plugin to receive these input logs. This part has been combined into the following examples.

##### Fluentd ClusterFluentdConfig example